command/genesis/params.go

```diff
@@ -384,6 +402,46 @@ func (p *genesisParams) predeployStakingSC() (*chain.GenesisAccount, error) {
 	return stakingAccount, nil
 }
 
+func (p *genesisParams) extractTokenParams() error {
+	if p.nativeTokenConfigRaw == "" {
+		p.nativeTokenConfig = &polybft.TokenConfig{
+			TokenName:     defaultTokenName,
+			TokenSymbol:   defaultTokenSymbol,
+			TokenDecimals: defaultTokenDecimals,
+		}
+	} else {
```

Comment:
Nitpick: 
consider early jumping here? Maybe slightly easier to follow instead of having a large else branch.

Note that I haven't removed the trailing bracket at the end of `else`.
```suggestion
	if p.nativeTokenConfigRaw == "" {
		p.nativeTokenConfig = &polybft.TokenConfig{
			TokenName:     defaultTokenName,
			TokenSymbol:   defaultTokenSymbol,
			TokenDecimals: defaultTokenDecimals,
		}
		
		return nil
	}
```